### ISSUE TITLE
Not able to set a title for _index.md

### ISSUE THREAD

**Issue by eisbehr-** [justinwalsh/daux.io#456]
I have an `_index.md` file in `docs/` folder as entry point. It seems like there is no way to add a page title. It is always empty and only showing the separator line. Is there anything I can do to have an page title on that index file?

I've already tried to use front matter. But that seems to make no difference.
```TEXT
---
title: Test
---
```

**_index.md file without title**
![no_title](https://cloud.githubusercontent.com/assets/917005/26058263/68fc7d50-397d-11e7-92aa-4ef8769d060c.PNG)

**Another, normal document viewing the page title:**
![title](https://cloud.githubusercontent.com/assets/917005/26058264/6900e6ec-397d-11e7-8500-90b0669f1e42.PNG)

**Comment by onigoetz** [comment]
Hello, what other options did you use ? 
Normally the main page is presented as a landing page which doesn't need a title.

**Comment by eisbehr-** [comment]
I don't want to have the lading page. So I set `auto_landing : false` and created an `_index.md` file as my entry point in docs. This works great so far, the only thing here is that I'm not able to set a title ...

I've already tested to use `index.md` instead of `_index.md`, what makes no difference.

ANother hint, using `front matter` to set the title workes perfectly for the browsers page title. But it wound be set as title of the document.

Content of `_index.md`:
```TEXT
---
title: Test !!
---
test
```

Output, Browsertitle is correct, heading is missing:
![title](https://cloud.githubusercontent.com/assets/917005/26092090/4bea9fc0-3a0f-11e7-95eb-3cf33eebae81.PNG)
